fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -118,13 +109,20 @@ object FaciaContent {
     linkSnap => Option("LinkSnap"),
     latestSnap => Option("LatestSnap"))
 
-  def imageReplace(fc: FaciaContent): Option[ImageReplace] = fold(fc)(
-    curatedContent => curatedContent.imageReplace,
-    supportingCuratedContent => supportingCuratedContent.imageReplace,
+  def image(fc: FaciaContent): Option[FaciaImage] = fold(fc)(
```

Comment:
I'm not so sure about this name because one might think it also incorporates the image from `Content`, which it doesn't. Can you think of a better name?

`maybeImageFromTool`?